src/pokemon_jump.c

```diff
@@ -1808,6 +1816,21 @@ static void sub_802C280(void)
         PlaySE(SE_DANSA);
 }
 
+static const s8 gUnknown_082FB65C[][48] =
+{
+    {-3, -6, -8, -10, -13, -15, -17, -19, -21, -23, -25, -27, -28, -29, -30, -30, -30, -28, -27,
+    -26, -25, -23, -22, -20, -18, -17, -15, -13, -11, -8, -6, -4,
+    -1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0},
```

Comment:
Trailing 0s are not necessary as the compiler will pad with 0s to the end of the allocation.